app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+  def resource_name
+    :user
```

Comment:
not really good approach, as all methods from all helpers overrides each other upon including

mb try to add to controller instead?